src/main/java/wooteco/subway/exception/badrequest/BadRequestException.java

```diff
@@ -0,0 +1,11 @@
+package wooteco.subway.exception.badrequest;
+
+import org.springframework.http.HttpStatus;
+import wooteco.subway.exception.SubwayException;
+import wooteco.subway.exception.dto.ErrorResponse;
+
+public class BadRequestException extends SubwayException {
```

Comment:
```suggestion
public abstract class BadRequestException extends SubwayException {
```

BadRequestException은 상속으로만 사용하고 있는데, 계층을 위해서만 사용한다면 abstract를 통해 의도를 전달할 수 있습니다 :)
추가적으로 저는 모든 클래스를 abstract 또는 final로만 작성하는데 그 이유에 대해 고민해보면 좋을 것 같아요 ㅎㅎ

Reply:
훨씬 좋은 거 같아요!!!! 👍👍👍👍👍
피드백대로 수정하고 나니 클래스에 대한 의도가 더 명확히 보이네요!!! 감사합니당!!